pages/common/gh-gist.md

```diff
@@ -3,22 +3,22 @@
 > Work with GitHub Gists on the command-line.
 > More information: <https://cli.github.com/manual/gh_gist>.
 
-- Create a new Gist from a space-separated list of files:
+- Create a new Gist from specific files:
 
-`gh gist create {{path/to/files}}`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}}`
```

Comment:
What does `(1..)` mean?

Reply:
It means that 1 or more file paths expected instead of 0 or more.

Reply:
Fixed. Such weird syntax was removed as leads to confusion.